src/components/Login.vue

```diff
@@ -12,9 +12,25 @@ export default {
   name: 'login',
   data () {
     return {
-      error: ''
+      error: '',
+      gapi: undefined
     }
   },
+  mounted() {
+    if (auth.isAuthorized()) {
+      this.$router.replace(this.$route.query.redirect || '/')
+    }
+    if (!window.gapi) {
+      return;
+    } else {
+      this.gapi = window.gapi;
+    }
+
+    gapi.signin2.render('g-signin2', {
```

Comment:
Does `gapi` resolve here?

Reply:
Like should it be `this.gapi`

Reply:
`gapi` is a global that gets added to window when the `<script src="https://apis.google.com/js/platform.js" async defer></script>` loads